components/CreateCycle/ExpandableWorkoutCard.jsx

```diff
@@ -74,14 +89,13 @@ const ExpandableWorkoutCard = (props) => {
           <SubTitle>Previous</SubTitle>
         </RowHeader>
         <FlatList
-          data={exercises}
-          // TODO: rename to exercise name
+          data={parsedExercises}
           keyExtractor={(item) => item.id}
           renderItem={({ item }) => (
             <RowContent>
-              <WorkoutViewText style={{ marginLeft: 10 }}>{item.id}</WorkoutViewText>
+              <WorkoutViewText style={{ marginLeft: 10 }}>{item.name}</WorkoutViewText>
               {/* TODO: center the sets x reps around the x so double digit numbers dont look bad */}
```

Comment:
Aye @finlaylp do you remember what this todo was referring to?

Reply:
yeah that was so that like 3 x 12 and 3 x 8 would be aligned around the x instead of just centered, bc it looks a bit odd that the x's can sometimes be offset a bit. Not a big issue but I didn't get around to it.